tmate-ssh-server.c

```diff
@@ -473,7 +495,28 @@ void tmate_ssh_server_main(struct tmate_session *session, const char *keys_dir,
 	if (!client->session)
 		tmate_fatal("Cannot initialize session");
 
+	fds[0].fd = ssh_bind_get_fd(daemon_bind);
+	fds[0].events = POLLIN;
+	fds[1].fd =  ssh_bind_get_fd(client_bind);
+	fds[1].events = POLLIN;
+
 	for (;;) {
+		ret = poll(fds, 2, 10 * 1000);
```

Comment:
minor: you can set the timeout to -1 to make it more obvious that you will wait forever anyway.